starter-dev-e2e/cypress/support/e2e.ts

```diff
@@ -1,13 +1,10 @@
 import "./commands.ts";
-import { View } from "../utils/view";
 
 beforeEach(() => {
   cy.intercept(/.*\/auth.token/, {
     fixture: "auth/token.json",
   }).as("token");
   cy.mockNextAuthCookie();
   cy.mockRemixAuthCookie();
-  cy.interceptGraphQLCalls(View.Dashboard);
-  cy.interceptRestCalls(View.Dashboard);
```

Comment:
Confirmed this isn't needed -- in `dashboard.cy.ts` in the `beforeEach` let's add the following on line 8: `cy.visit("/");`

This will cause calls that were failing on the redirect to this page in this `e2e` file to be recalled and allow the flaky intercepts due to a race condition happen reliably!